test/test_markdown2.py

```diff
@@ -166,7 +166,9 @@ def charreprreplace(exc):
                           _display(python_toc_html), _display(toc_html),
                           _indent(diff))
             self.assertEqual(python_norm_toc_html, norm_toc_html,
-                errmsg.encode('ascii', 'charreprreplace'))
+                errmsg.encode('utf-8', 'charreprreplace'))
+            # self.assertEqual(python_norm_toc_html, norm_toc_html,
+            #     errmsg.encode('ascii', 'charreprreplace'))
```

Comment:
I think it's right. Should be utf-8. I'm not too worried as this is just the test code anyway.